modules/nf-core/tcoffee/consensus/main.nf

```diff
@@ -0,0 +1,62 @@
+process TCOFFEE_CONSENSUS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/t-coffee_pigz:f47b85d70360f1a0':
+        'community.wave.seqera.io/library/t-coffee_pigz:6c9b2f8b97ee55e5' }"
+
+
+    input:
+    tuple val(meta), path(aln)
+    tuple val(meta2), path(tree)
+    val(compress)
+
+    output:
+    tuple val(meta), path("*.aln{.gz,}"), emit: alignment
```

Comment:
```suggestion
    tuple val(meta), path("*.aln{.gz}"), emit: alignment
```

did you forget another file type here maybe ?

Reply:
Ah ups no, just a typo!